joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/ControlStructureTests.scala

```diff
@@ -337,27 +337,29 @@ class ControlStructureTests extends RubyCode2CpgFixture {
         |end
         |""".stripMargin)
 
-    val List(rescueNode) = cpg.method("test1").tryBlock.l
-    rescueNode.controlStructureType shouldBe ControlStructureTypes.TRY
-    val List(body, rescueBody1, rescueBody2, rescueBody3, elseBody, ensureBody) = rescueNode.astChildren.l
-    body.ast.isLiteral.code.l shouldBe List("1")
-    body.order shouldBe 1
+    inside(cpg.method("test1").controlStructure.l) {
+      case tryStruct :: rescue1Struct :: rescue2Struct :: rescue3Struct :: elseStruct :: ensureStruct :: Nil =>
+        tryStruct.controlStructureType shouldBe ControlStructureTypes.TRY
+        val List(body, _, _, _, _, _) = tryStruct.astChildren.l
```

Comment:
Unless you're intentionally testing the number of children, this might be simpler
```suggestion
        val body = tryStruct.astChildren.head
```